CONTRIBUTING.md

```diff
@@ -0,0 +1,13 @@
+# Getting Started
+
+## Installation
+
+```bash
+pnpm i
+```
+
+## Development
+
+```bash
+pnpm dev
```

Comment:
I still need to figure out how to get this to work properly, some ports need to be remapped, probably in the examples repo.